pkg/apis/operator/v1alpha1/knativeserving_types.go

```diff
@@ -53,9 +53,11 @@ type KnativeServingSpec struct {
 	CommonSpec `json:",inline"`
 
 	// A means to override the knative-ingress-gateway
+	// Deprecated: Use Ingress.Istio.KnativeIngressGateway instead.
```

Comment:
I agree with the plan of deprecation. Just suggest we keep them at least till 0.16.
Hope in 15, we have KnativeIngressGateway and ClusterLocalGateway in here and Ingress.
For 16 maybe, we use conversion webhook to transform old CR to new CR.